### PR TITLE
(maint) Update puppet5-nightly repo paths

### PR DESCRIPTION
**Pull request by mwaggett**

### CODE REVIEW

File: configs/components/gpg_key.rb [hunk 1]
```diff
@@ -5,10 +5,8 @@
     pkg.add_source 'file://files/puppet5-nightly-keyring.gpg'
     pkg.install_file 'puppet5-nightly-keyring.gpg', '/etc/apt/trusted.gpg.d/puppet5-nightly-keyring.gpg'
   else
-    pkg.add_source("file://files/RPM-GPG-KEY-nightly-puppetlabs")
     pkg.add_source("file://files/RPM-GPG-KEY-puppet.asc")
-    pkg.install_file 'RPM-GPG-KEY-nightly-puppetlabs', '/etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-nightly'
-    pkg.install_file 'RPM-GPG-KEY-puppet.asc', '/etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-release'
```
Comment (shrug) [inline configs/components/gpg_key.rb]: You still need this line to install the key, no?

File: configs/components/gpg_key.rb [hunk 2]
```diff
@@ -5,10 +5,8 @@
...
+    pkg.install_file 'RPM-GPG-KEY-puppet.asc', '/etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-nightly-release'
```
Comment (mwaggett) [inline configs/components/gpg_key.rb]: @shrug it's still here, I just changed the install location.

File: files/puppet5-nightly.repo.txt [hunk 3]
```diff
@@ -1,14 +1,14 @@
 [puppet5-nightly]
 name=Puppet 5 Nightly Repository __OS_NAME__ __OS_VERSION__ - $basearch
-baseurl=http://yum.puppetlabs.com/puppet5-nightly/__OS_NAME__/__OS_VERSION__/$basearch
+baseurl=http://nightlies.puppet.com/yum/puppet5-nightly/__OS_NAME__/__OS_VERSION__/$basearch
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-nightly
```
Comment (shrug) [inline files/puppet5-nightly.repo.txt]: We can lose the "nightly" key here

File: files/puppet5-nightly.repo.txt [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
        file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-release
 enabled=1
 gpgcheck=1
 
 [puppet5-nightly-source]
 name=Puppet 5 Nightly Repository __OS_NAME__ __OS_VERSION__ - Source
-baseurl=http://yum.puppetlabs.com/puppet5-nightly/__OS_NAME__/__OS_VERSION__/SRPMS
+baseurl=http://nightlies.puppet.com/yum/puppet5-nightly/__OS_NAME__/__OS_VERSION__/SRPMS
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-nightly
        file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-release
```
Comment (shrug) [inline files/puppet5-nightly.repo.txt]: We can lose the "nightly" key here

File: configs/components/gpg_key.rb (hunk 2 above)
Comment (shrug) [reply configs/components/gpg_key.rb]: doh!

File: files/puppet5-nightly.repo.txt [hunk 5]
```diff
@@ -1,16 +1,14 @@
...
 enabled=1
 gpgcheck=1
 
 [puppet5-nightly-source]
 name=Puppet 5 Nightly Repository __OS_NAME__ __OS_VERSION__ - Source
-baseurl=http://yum.puppetlabs.com/puppet5-nightly/__OS_NAME__/__OS_VERSION__/SRPMS
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-nightly
-       file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-release
+baseurl=http://nightlies.puppet.com/yum/puppet5-nightly/__OS_NAME__/__OS_VERSION__/SRPMS
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet5-nightly-release
```
Comment (mwaggett) [inline files/puppet5-nightly.repo.txt]: @shrug updated

**Pull request merged**